README.md

```diff
@@ -199,6 +214,30 @@ bin/dev
 ```
 This starts css/js bundling and the development server
 
+### Generating local technical metadata locally
+
+If you have a file on your laptop you want to test quickly to see if generation is as expected, you can do this on the rails console.
+
+```ruby
+rails c
+
+druid = 'druid:ab123bc4567'
+basepath = '//some/local/laptop/path'
+files = [{"uri":"file:///some/local/laptop/path/nc889qn3957_sl.mp4", "md5": "012"}]
+params = {druid:, files:, basepath:}
+file_infos = params[:files].map do |file|
+    filepath = CGI.unescape(URI(file[:uri]).path)
+    filename = FilepathSupport.filename_for(filepath:, basepath: params[:basepath])
+    FileInfo.new(filepath:, md5: file[:md5], filename:)
+end
+errors = TechnicalMetadataGenerator.generate_with_file_info(druid:, file_infos:, force: true)
+
+puts errors
+
+pp DroFile.where(druid:)
+DroFile.where(druid:).each {|file| pp file.dro_file_parts};nil
+```
```

Comment:
this was helpful for me when i was developing as it bypassed jobs and workflow, etc. and just went ahead and created the tech metadata immediately